idxmgmt/manager.go

```diff
@@ -225,22 +224,26 @@ func (m *manager) loadEventTemplate(feature feature, ilmSupporter libilm.Support
 	return nil
 }
 
-func (m *manager) loadPolicy(ilmFeature feature, ilmSupporter libilm.Supporter, policiesLoaded []string) ([]string, error) {
+func (m *manager) loadPolicy(ilmFeature feature, ilmSupporter libilm.Supporter, policiesLoaded map[string]bool) error {
+	if !ilmFeature.enabled {
+		return nil
+	}
 	policy := ilmSupporter.Policy().Name
-	if !ilmFeature.enabled || utility.Contains(policy, policiesLoaded) {
-		return policiesLoaded, nil
+	if policiesLoaded[policy] {
```

Comment:
Why are you removing the `!ilmFeature.enabled` check here?

Reply:
It's not removed, I just separated the check above before calling `ilmFeature.Policy()`

Reply:
two lines up 🤦‍♀️ - thanks